### PR TITLE
Fix table dimensions to match Figma

### PR DESCRIPTION
**Pull request by XeniaRud**

### CODE REVIEW

**Comment by duci9y** [comment]
God bless, what a thing of beauty.

File: client/src/pages/MemberDashboard/MemberDashboard.module.css [hunk 1]
```diff
@@ -3,19 +3,115 @@
   justify-content: space-between;
   align-items: center;
   padding: 0 20px;
```
Comment (duci9y) [inline client/src/pages/MemberDashboard/MemberDashboard.module.css:5]:
```suggestion
```

File: client/src/pages/MemberDashboard/MemberDashboard.module.css [hunk 2]
```diff
@@ -2,20 +2,115 @@
...
+}
+
+.tableClass th {
+  margin-bottom: 10px;
+  text-align: left;
   background: #eceff8;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 12px;
```
Comment (controversial) [inline client/src/pages/MemberDashboard/MemberDashboard.module.css:57]:
```suggestion
  font-size: 12px;
  padding: 0 16px;
```

File: client/src/pages/MemberDashboard/MemberDashboard.module.css [hunk 3]
```diff
@@ -3,19 +3,115 @@
...
-.projectreq {
-  background: #fff;
+.header2 {
+  margin: 30px;
 }
 
-.reqtable {
-  margin-bottom: 10px;
+.searchBar {
+  float: right;
```
[the remark this answers is not in the document]
Comment (XeniaRud) [reply client/src/pages/MemberDashboard/MemberDashboard.module.css:21]: When I tried to change it to a flexbox, align-self did not work.

[the remark this answers is not in the document]
Comment (duci9y) [reply client/src/pages/MemberDashboard/MemberDashboard.module.css]: I fixed it, feel free to take a look at the code and let me know if it feels wrong!

**Pull request merged**